lib/core/Manager.js

```diff
@@ -112,7 +114,12 @@ Manager.prototype.resolve = function () {
     // Otherwise, fetch each target from the repository
     // and let the process roll out
     } else {
-        this._targets.forEach(this._fetch.bind(this));
+        this._targets.forEach(function (target) {
+            var name = target.name || target.source;
```

Comment:
Can you define the `var` outside the loop and just reuse it later in each iteration?